x/erc20/keeper/integration_test.go

```diff
@@ -195,50 +195,39 @@ var _ = Describe("ERC20:", Ordered, func() {
 	})
 
 	Describe("Converting", func() {
-		Context("with a registered coin", func() {
+		Context("with a registered ERC20", func() {
 			BeforeEach(func() {
-				pair = s.setupRegisterCoin(metadataCoin)
+				contract := s.setupRegisterERC20Pair(contractMinterBurner)
+				id := s.app.Erc20Keeper.GetTokenPairID(s.ctx, contract.String())
+				pair, _ = s.app.Erc20Keeper.GetTokenPair(s.ctx, id)
 				coin = sdk.NewCoin(pair.Denom, amt)
 
 				err := testutil.FundAccount(s.ctx, s.app.BankKeeper, accAddr, sdk.NewCoins(sdk.NewCoin(utils.BaseDenom, fundsAmt)))
 				s.Require().NoError(err)
-				err = testutil.FundAccount(s.ctx, s.app.BankKeeper, accAddr, sdk.NewCoins(coin))
-				s.Require().NoError(err)
+
+				_ = s.MintERC20Token(contract, s.address, addr, big.NewInt(amt.Int64()))
+				s.Commit()
 			})
-			Context("with a registered ERC20", func() {
+
+			Describe("an ERC20 token into a Cosmos coin", func() {
 				BeforeEach(func() {
-					contract := s.setupRegisterERC20Pair(contractMinterBurner)
-					id := s.app.Erc20Keeper.GetTokenPairID(s.ctx, contract.String())
-					*pair, _ = s.app.Erc20Keeper.GetTokenPair(s.ctx, id)
-					coin = sdk.NewCoin(pair.Denom, amt)
+					convertERC20(s.ctx, s.app, privKey, amt, pair.GetERC20Contract())
+				})
```

Comment:
are we going to remove this method too? That would be more in line with us moving away from manual conversions with the introduction of STR v2